runtime/include/localeModels/flat/chpl-locale-model.h

```diff
@@ -66,4 +66,17 @@ c_sublocid_t chpl_rt_sublocFromLocaleID(chpl_localeID_t loc) {
   return c_sublocid_any;
 }
 
+//
+// These functions are exported from the locale model for use by
+// the tasking layer to convert between a full sublocale and an
+// execution sublocale.
+//
+extern
+c_sublocid_t chpl_localeModel_sublocToExecutionSubloc(
```

Comment:
Ah.  Now I see why these have the `chpl_localeModel_` prefix.  I still think there's a simpler way.  Can the tasking layers just understand low-level sublocales, what I think you're calling "execution" sublocales, and the module code deal with any need to turn these into more-descriptive things?  I.e., make the sublocToExecutionSubloc() and sublocMerge() calls in the module code that interacts with the C runtime, rather than in the C runtime itself?